pkgs/tools/security/please/default.nix

```
@@ -8,16 +8,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "please";
```

I believe Repology uses the pname
```suggestion
  pname = "pleaser";
```